contrib/logtesting/dotests.sh

```diff
@@ -19,17 +25,22 @@ for i in ./*/log; do
     MD2=`md5sum $idir/res | cut -d " " -f 1`
 
     if [ ! $MD1 = $MD2 ]; then
+        exitcode=1
+        echo
+        echo
+        echo
         echo "**ERROR: Unit testing failed. Output for the test $i failed."
-        echo "== OLD OUTPUT: =="
-        cat $idir/res
-	echo "== NEW OUTPUT: =="
-        cat tmpres
-        echo "** ERROR: Exiting."
+        echo "== DIFF OUTPUT (reference | new): =="
+        diff -Nay $idir/res tmpres
```

Comment:
My `diff` doesn't have a `-y` flag.

Reply:
I checked with the virtual ossec appliance and the ``-y `` exists...
What is your distribution ?

Reply:
OpenBSD in this case. I haven't checked others.